helloworld-shell/Dockerfile

```diff
@@ -44,6 +44,9 @@ RUN set -x && apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -
     ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
+# Create and change to the app directory.
+WORKDIR /app
```

Comment:
not needed

Reply:
Did the linter change? AFAICT this is exactly what `hadolint` was failing on.

Reply:
I didn't see the second FROM statement. This warning is coming from a new version of hadolint. To address this issue we need to think about pinning to a version. Also please update this to `WORKDIR /`. The workdir /app is confusing with the copy statements below